<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;

border-image: url(:/image/image/background 700x400.png);

border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tools">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(28, 28, 28);
border:none;
border-top-left-radius:30px;
border-bottom-left-radius:30px;
border-right:5px solid rgba(70,70,70,255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>296</y>
     <width>52</width>
     <height>84</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="showmini">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>34</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton:hover {
        icon: url(:/icon/icon/showmini_hover.png);
    }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../pic/res.qrc">
      <normaloff>:/icon/icon/showmini.png</normaloff>:/icon/icon/showmini.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>45</y>
      <width>34</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton:hover {
        icon: url(:/icon/icon/close_hover.png);
    }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../pic/res.qrc">
      <normaloff>:/icon/icon/close.png</normaloff>:/icon/icon/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>9</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 0);
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_api_0">
    <widget class="QLineEdit" name="api_search_bvid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;猫啃什锦黑-轻量版&quot;;
border:none;
border-bottom:5px solid rgba(255,255,255,255);
background-color: rgba( 0, 0, 0, 0);
color: rgb(255,255,255);</string>
     </property>
     <property name="placeholderText">
      <string>Video Bvid</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>82</width>
       <height>280</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QPushButton" name="view_ui_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>64</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../pic/res.qrc">
        <normaloff>:/icon/icon/view.png</normaloff>:/icon/icon/view.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="like_ui_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>64</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../pic/res.qrc">
        <normaloff>:/icon/icon/like.png</normaloff>:/icon/icon/like.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="coin_ui_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>64</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../pic/res.qrc">
        <normaloff>:/icon/icon/coin.png</normaloff>:/icon/icon/coin.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="collect_ui_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>64</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../pic/res.qrc">
        <normaloff>:/icon/icon/collect.png</normaloff>:/icon/icon/collect.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>71</width>
       <height>280</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QPushButton" name="share_ui_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>64</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../pic/res.qrc">
        <normaloff>:/icon/icon/share.png</normaloff>:/icon/icon/share.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="reply_ui_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>64</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../pic/res.qrc">
        <normaloff>:/icon/icon/reply.png</normaloff>:/icon/icon/reply.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="danmaku_ui_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>64</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../pic/res.qrc">
        <normaloff>:/icon/icon/danmaku.png</normaloff>:/icon/icon/danmaku.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="vt_ui_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>64</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../pic/res.qrc">
        <normaloff>:/icon/icon/vt.png</normaloff>:/icon/icon/vt.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="view_text_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>114514000 View</string>
     </property>
    </widget>
    <widget class="QLabel" name="like_text_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>114514 Like</string>
     </property>
    </widget>
    <widget class="QLabel" name="coin_text_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>114514 Coin</string>
     </property>
    </widget>
    <widget class="QLabel" name="collect_text_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>330</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>114514 Collect</string>
     </property>
    </widget>
    <widget class="QLabel" name="share_text_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>120</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>114514 Share</string>
     </property>
    </widget>
    <widget class="QLabel" name="reply_text_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>190</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>114514 Reply</string>
     </property>
    </widget>
    <widget class="QLabel" name="danmaku_text_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>114514 Danmaku</string>
     </property>
    </widget>
    <widget class="QLabel" name="vt_text_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>330</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>114514 Min</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_video">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:none;
}
QPushButton:hover{
	padding-bottom:5px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../pic/res.qrc">
       <normaloff>:/icon/icon/search_button.png</normaloff>:/icon/icon/search_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_hub_1">
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>82</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="sub_ui">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../pic/res.qrc">
          <normaloff>:/icon/icon/sub.png</normaloff>:/icon/icon/sub.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="view_ui">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../pic/res.qrc">
          <normaloff>:/icon/icon/view.png</normaloff>:/icon/icon/view.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="like_ui">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../pic/res.qrc">
          <normaloff>:/icon/icon/like.png</normaloff>:/icon/icon/like.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="User_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;猫啃什锦黑-轻量版&quot;;
</string>
     </property>
     <property name="text">
      <string>闲踏梧桐</string>
     </property>
    </widget>
    <widget class="QLabel" name="like_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>暂不开放 Like</string>
     </property>
    </widget>
    <widget class="QLabel" name="view_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>暂不开放 View</string>
     </property>
    </widget>
    <widget class="QLabel" name="sub_text">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(184, 184, 184);
font: 20pt &quot;猫啃什锦黑-轻量版&quot;;</string>
     </property>
     <property name="text">
      <string>114514 Sub</string>
     </property>
    </widget>
    <widget class="QLabel" name="proverbs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>77</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;猫啃什锦黑-轻量版&quot;;
color: rgb(255, 255, 255);
letter-spacing: 3px;</string>
     </property>
     <property name="text">
      <string>——等待每一个盛大登场的机会</string>
     </property>
    </widget>
    <widget class="QLabel" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>600</width>
       <height>5</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(143, 143, 143);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>600</width>
       <height>5</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(143, 143, 143);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>13</y>
     <width>52</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="backhub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>34</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton:hover {
        icon: url(:/icon/icon/hub_hover.png);
    }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../pic/res.qrc">
      <normaloff>:/icon/icon/hub.png</normaloff>:/icon/icon/hub.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton:hover {
        icon: url(:/icon/icon/search_hover.png);
    }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../pic/res.qrc">
      <normaloff>:/icon/icon/search.png</normaloff>:/icon/icon/search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../pic/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>showmini</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>-39</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>-48</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
